src/components/layout/Footer.tsx

```
@@ -0,0 +1,62 @@
+                        <Link to="">Work</Link>
+                        <Link to="">Process</Link>
+                        <Link to="">About</Link>
+                        <Link to="">Careers</Link>
+                        <Link to="">Contact</Link>
+                    </div>
+                    {/* Right: Social */}
+                    <div className='flex flex-shrink-0 gap-[20px] items-center border rounded-[12px] border-[#262626] p-[14px] pl-[24px]'>
+                        <p>Stay Connected</p>
+                        <div className='flex gap-[14px] text-theme-white'>
+                            <Link to="" className='p-[20px] border rounded-[8px]'>
+                                <Facebook className='h-[24px] w-[24px]' />
```

ESLint

[error] Replace `································<Facebook·className='h-[24px]·w-[24px]'` with `················<Facebook·className="h-[24px]·w-[24px]"`

(prettier/prettier)